community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/RecordFormats.java

```diff
@@ -58,5 +68,22 @@ public Factory( String key, String... altKeys )
 
     RecordFormat<RelationshipTypeTokenRecord> relationshipTypeToken();
 
-    RecordFormat<RelationshipGroupRecord> relationshipGroup();
+    RecordFormat<DynamicRecord> dynamic();
+
+    boolean hasStore( StoreType store );
```

Comment:
do not think this method has something to do with RecordFormats, since we have no notions of stores here and should not operate them